src/components/UI/QuestionTextarea.vue

```diff
@@ -24,6 +24,10 @@ export default defineComponent({
       type: Boolean,
       default: false
     },
+    pointerevents: {
+      type: Boolean,
+      defaulet: false
```

Comment:
```suggestion
      default: false
```
この`pointerevents`ってどういう意図で付けました？